kivy/core/audio/audio_sdl2.pyx

```diff
@@ -42,7 +43,7 @@ cdef mix_init():
         mix_is_init = -1
         return 0
 
-    mix_flags = Mix_Init(0)
+    mix_flags = Mix_Init(MIX_INIT_FLAC|MIX_INIT_MOD|MIX_INIT_MP3|MIX_INIT_OGG)
```

Comment:
If you're missing support for one of these formats, does `Mix_Init` error out or does it just ignore the format?